cmd/interceptors/main.go

```diff
@@ -68,7 +68,8 @@ func main() {
 
 	service, err := server.NewWithCoreInterceptors(kubeClient, logger)
 	if err != nil {
-		log.Fatalf("failed to initialize core interceptors: %s", err)
+		log.Printf("failed to initialize core interceptors: %s", err)
+		return
```

Comment:
I don't think this change is meant to be in the PR, I wonder it the cherry-pick picked up too much?

Reply:
we need this else the build tests fail due to gocritic issues 😢

Reply:
Yeah exactly, we found this out in the v0.12.x patch (#1111). So there's 0fd25a6 to fix the titles, cd80b0b to please gocritic, and 6ddf136 to bump the plumbing version for integration tests to pass.